ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -232,7 +266,21 @@ export function AlertModal({
           />
         </ModalBody>
         <ModalFooter>
-          <AcknowledgeButton onAcknowledgeClick={onAcknowledgeClick} />
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            gap={3}
+            width={BlockSize.Full}
+          >
+            <AcknowledgeButton onAcknowledgeClick={onAcknowledgeClick} />
```

Comment:
If this is the primary button, should this go at the top or bottom? Can't recall what the Figma showed.

Reply:
I double-checked the figma and `Got it` is always on top but when we have more actions it should become a secondary button. I applied the necessary changes to support it.